### ISSUE TITLE
Flashfire back to a 1.5 second cast

### ISSUE THREAD

**Issue by swats-on** [WarEmu/WarBugs#15631]
Expected that Flashfire should cause next ability with a build uop time to cast instatntly.
Unexpected that the abiltiy has a 1.5 second (half) build up time instead.

I think this was previously fixed so perhaps it has been purposfully returned, if so tooltip needs update only.

Thanks for your continued efforts.

**Comment by SilverWF** [comment]
Yes, sometimes it's bugs out and instead of insta cast it' just halves cast time and allows to cast on the move.
